src/components/BookList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+const BookList = ({books}) => {
+  console.log(books)
+  return (
+    <ul>
+      {books.map((book) => (
+        <li key={book.id}>
+            {book.title} {book.author}
+        </li>
+      ))}
```

Comment:
- [Optional] Please add a button to remove the book from the list. The button functionality shouldn't be implemented yet but the markup should be in your file.

Reply:
@Kamba56 Thank you for suggestions and review. I will be applying both of them in my next project for this repository.